lib/wings/services/custom_queries/find_file_metadata.rb

```diff
@@ -1,11 +1,11 @@
 # frozen_string_literal: true
-# Custom query override specific to Wings for finding Hydra::PCDM::File and converting to Hyrax::FileMetadata.
-# @example
-#   Hyrax.custom_queries.find_file_metadata_by(id: valkyrie_id, use_valkyrie: true)
-#   Hyrax.custom_queries.find_file_metadata_by_alternate_identifier(alternate_identifier: id, use_valkyrie: true)
-require 'wings/services/file_converter_service'
 module Wings
   module CustomQueries
+    # Custom query override specific to Wings for finding Hydra::PCDM::File and converting to Hyrax::FileMetadata.
```

Comment:
this class is largely rewritten, since the basic Wings components now do most of its work. `FileConverterService` is take out of service completely (and deprecated). there are also refactors here. changes are separated across several commits to keep refactors and functional changes apart. see commit messages for more.